docs/deployment/DeploymentGuide.md

```diff
@@ -0,0 +1,89 @@
+## Setting up a development system
+
+The following things must be installed in order to work with the development version of the program,
+and to create packaged, installable executables for distribution to others.
+
+In general, the version requirements are stricter on Windows than on Mac OS, due to the difficulty of
+compiling binaries on Windows.
+
+Separate MacOS setup instructions are available and will be added here at a later date.
+
+
+- `Python` >= 3.4
+  - The program will run on Python 3.5+, but [3.4.x](https://www.python.org/downloads/release/python-343/) is
+  recommended for Windows operating systems, for best compatibility with precompiled Windows PyQT binaries.
+  - On Windows, be sure to select a custom install, and choose the option to add Python (and script folders)
+  to the system PATH automatically.
+- `pip`
+  - Should already come with python. Make sure it can be run via the command line.
+- `git`
+  - On Windows, the [GitHub desktop client](https://desktop.github.com/) provides a viable installation. (default
+  data folder is `My Documents\GitHub`)
+- [qt5.5](http://www.qt.io/download-open-source/)
+  - Can de-select QT 5.4 from the installer to save disk space; only need 5.5. Other options left as-is.
```

Comment:
Suggest to de-select 5.4 in case the wrong configure will cause the usage of 5.4 instead of 5.5 which has a problem with system tray icon disappear sometime.